xdis/opcodes/opcode_311.py

```diff
@@ -119,28 +119,12 @@
 # match, these two ops had stack effects changed
 rm_op(loc,  "MATCH_CLASS",            152)
 rm_op(loc,  "MATCH_KEYS",              33)
-
-## Redefined OPS.
-# Be careful to redefine in a way that that does not rm_op() an opcode
-# *after* the new one with that opcode has been (re)entered. For example,
-# we to redefine 138 from DELETE_REF to STORE_DEREF.
-# So we have to rm DELETE_DEREF with opcode 138 *before* adding
-# STORE_DEREF with opcode 138.
-
```

Comment:
Isn't this comment still valid? If so it can be moved down below before the def operations.

Reply:
Sorry about that, yes that comment is still useful.